json-lint/test/resources/schema/inventory/valid/engageya.test.json

```diff
@@ -0,0 +1,18 @@
+{
+    "placements": {
+        "nativeAd": {
+            "adServer": "engageya"
+        }
+    },
+    "adServers": {
+        "engageya": {
+            "data-widgetIds": "108404",
```

Comment:
int

Reply:
In the documentation could be a list of ids separated by comma (id1,id2)